sdk/src/test/java/io/dapr/client/DaprHttpTest.java

```diff
@@ -42,6 +47,38 @@ public void setUp() throws Exception {
     okHttpClient = new OkHttpClient.Builder().addInterceptor(mockInterceptor).build();
   }
 
+  @Test
+  public void invokeApi_daprApiToken_present() throws IOException {
+    mockInterceptor.addRule()
+        .post("http://127.0.0.1:3500/v1.0/state")
+        .hasHeader(Constants.DAPR_API_TOKEN_HEADER)
+        .respond(serializer.serialize(EXPECTED_RESULT));
+    environmentVariables.set(Constants.DAPR_API_TOKEN, "xyz");
```

Comment:
Do not set environment variable, set the property instead and the behavior should be the same.